### ISSUE TITLE
critic.t test may fail

### ISSUE THREAD

**Issue by eserte** [memowe/EventStore-Tiny#14]
On some of my smokers the t/0_qa/critic.t fails:
```
#   Failed test 'Test::Perl::Critic for "/usr/home/eserte/.cpan/build/2018062011/EventStore-Tiny-0.22-1/t/0_qa/../../lib/EventStore/Tiny.pm"'
#   at /usr/perl5.16.3t/lib/site_perl/5.16.3/Test/Perl/Critic.pm line 121.
# 
#   Method invocation should use "()" at line 25, column 40.  Use "->MethodName()" instead of "->MethodName".  (Severity: 5)
#   Method invocation should use "()" at line 27, column 49.  Use "->MethodName()" instead of "->MethodName".  (Severity: 5)
#   Variable, subroutine, and package names have to be in CamelCase at line 32, column 1.  (no explanation).  (Severity: 5)
#   Variable, subroutine, and package names have to be in CamelCase at line 33, column 16.  (no explanation).  (Severity: 5)
... etc. ...
```

Probably this depends on the existence of some additional perlcritic plugins.

**Comment by memowe** [comment]
Thanks, @eserte, I'll have a look at it!

**Comment by memowe** [comment]
Should be resolved by 2b48293 in release v0.3. Thanks! :heart:

**Comment by memowe** [comment]
I tried to define the critics test a little bit better in 2b48293 but I don't see how to prevent perlcritic to use additional modules I don't have. Any ideas?

**Comment by memowe** [comment]
Solved in 29f8c980 (release v0.3) by defining perlcritic tests as author-only, as recommended in [T::P::C](http://p3rl.org/Test::Perl::Critic). Thanks again!
